README.md

```diff
@@ -64,11 +54,18 @@ The config file can be JSON, TOML, YAML, HCL, or a Java properties file. Keys do
         "root": "toor"
     },
     "VIOLATION_PENALTIES": {
-        "rate-limit-exceeded": 2
+        "rate_limit_exceeded": 2
```

Comment:
I don't think this part here works still, this spec doesn't load (it's just parsed as an empty map) but if we change this to:

`"VIOLATION_PENALTIES": "rate_limit_exceeded=2"`

it works as expected.

Reply:
Good catch! 

Fixed in 885f2c0.

NB: the violation penalties config is a string since viper wouldn't parse a map with keys containing colons correctly (and the FxA integration sends violations with names like `fxa:blockIP:...`)